app/controllers/comments_controller.rb

```diff
@@ -46,6 +48,10 @@ def update
       if @comment.update(comment_params)
         format.html { redirect_to root_url, notice: "Comment was successfully updated." }
         format.json { render :show, status: :ok, location: @comment }
+        format.js 
+        # do
```

Comment:
Can delete unused code I see you kept it for learning purposes but you can always view previous commits